docs/util.md

```diff
@@ -49,5 +49,19 @@ let resultFalse = isvalid('88075290')
 // resultFalse => false
 ```
 
+## isURL
+
+Checks for url
+
+#### Usage
+```es6
+import {validate, util} from 'valid.js'
+
+let isvalid = validate(util.isURL)
+let result = isvalid('https://github.com')
+// result => true
+let resultFalse = isvalid('http://.google.com')
```

Comment:
In fact, it's a valid URL. The [regex](https://gist.github.com/dperini/729294) I used, is developed in order to validate the most common used urls (so this regex will consider `http://localhost/`, `http://localhost:8080/` as invalid).

Following is a comment from, the regex source code:
```js
// the regular expression composed & commented
// could be easily tweaked for RFC compliance,
// it was expressly modified to fit & satisfy
// these test for an URL shortener:
//
//   http://mathiasbynens.be/demo/url-regex
//
// Notes on possible differences from a standard/generic validation:
//
// - utf-8 char class take in consideration the full Unicode range
// - TLDs have been made mandatory so single names like "localhost" fails
// - protocols have been restricted to ftp, http and https only as requested
//
// Changes:
//
// - IP address dotted notation validation, range: 1.0.0.0 - 223.255.255.255
//   first and last IP address of each class is considered invalid
//   (since they are broadcast/network addresses)
//
// - Added exclusion of private, reserved and/or local networks ranges
//
// - Made starting path slash optional (http://example.com?foo=bar)
//
// - Allow a dot (.) at the end of hostnames (http://example.com.)
```